keymaps/intellij-idea-keymap.cson

```diff
@@ -230,6 +230,12 @@
   # Duplicate current line or selected block
   'ctrl-d': 'editor:duplicate-lines'
 
+  # Start new line
+  'shift-enter': 'editor:newline-below'
+
+  # Change selected text to upper-case
+  'ctrl-shift-u': 'editor:upper-case'
```

Comment:
Looks like I messed up my pull request, I intended them to be separate. Sorry, I'll fix it.